INFO - 2021-12-16 16:00:02,956 - new_apply_order.py[line:893]: 开始第 fakash【1】次进件……
INFO - 2021-12-16 16:00:02,957 - new_apply_order.py[line:84]: ======【ke测试环境】生成手机号：739641602 ======
INFO - 2021-12-16 16:00:03,347 - new_apply_order.py[line:97]: 739641602 检测用户存在成功！
INFO - 2021-12-16 16:00:03,416 - new_apply_order.py[line:106]: 739641602 短信发送成功！
INFO - 2021-12-16 16:00:03,477 - new_apply_order.py[line:119]: 739641602 注册成功！
INFO - 2021-12-16 16:00:03,586 - new_apply_order.py[line:135]: 739641602 登录短信发送成功！
INFO - 2021-12-16 16:00:03,646 - new_apply_order.py[line:145]: userId=6242779090255224832,mobile=739641602,token=7d26b4e7a8aa72081f8ce41f8a51e5bf 登录成功
INFO - 2021-12-16 16:00:03,779 - new_apply_order.py[line:212]: 个人信息提交成功！
INFO - 2021-12-16 16:00:03,847 - new_apply_order.py[line:232]: 联系人信息提交成功！
INFO - 2021-12-16 16:00:04,116 - new_apply_order.py[line:255]: 创建预审订单成功！
INFO - 2021-12-16 16:00:04,129 - new_apply_order.py[line:263]: 预审状态修改为通过,更新条数 1
INFO - 2021-12-16 16:00:04,138 - new_apply_order.py[line:266]: 更新初始额度！更新条数 1
INFO - 2021-12-16 16:00:04,397 - new_apply_order.py[line:342]: 活体图片上传成功！回写配置文件global_param.mp4_url
INFO - 2021-12-16 16:00:04,457 - new_apply_order.py[line:352]: 视频认证通过！
INFO - 2021-12-16 16:00:04,725 - new_apply_order.py[line:579]: 739641602 短信发送成功！
INFO - 2021-12-16 16:00:04,906 - new_apply_order.py[line:588]: 绑卡成功！
INFO - 2021-12-16 16:00:05,431 - new_apply_order.py[line:623]: 进件成功！执行sql将订单改为风控中……
INFO - 2021-12-16 16:00:06,467 - new_apply_order.py[line:629]: 订单表更新1条，信审更新1条
INFO - 2021-12-16 16:00:06,829 - new_apply_order.py[line:643]: 查询loan_order_id成功！
INFO - 2021-12-16 16:00:06,881 - new_apply_order.py[line:658]: 订单回调复审通过成功！
INFO - 2021-12-16 16:00:07,691 - new_apply_order.py[line:680]: 获取token成功！token=bearer c69d1a29-d8cd-4a53-a299-3fc02203f2e4！
INFO - 2021-12-16 16:00:08,053 - new_apply_order.py[line:643]: 查询loan_order_id成功！
INFO - 2021-12-16 16:00:08,418 - new_apply_order.py[line:698]: 后台已触发放款成功！
INFO - 2021-12-16 16:00:08,430 - new_apply_order.py[line:712]: xxl_job web 登录成功！
INFO - 2021-12-16 16:00:08,439 - new_apply_order.py[line:728]: xxl_job 放款准备作业执行成功！
INFO - 2021-12-16 16:00:09,800 - new_apply_order.py[line:643]: 查询loan_order_id成功！
INFO - 2021-12-16 16:00:09,818 - new_apply_order.py[line:736]: 修改放款准备完成时间成功！
INFO - 2021-12-16 16:00:09,825 - new_apply_order.py[line:744]: xxl_job 放款完成作业执行成功！
INFO - 2021-12-16 16:00:10,184 - new_apply_order.py[line:643]: 查询loan_order_id成功！
INFO - 2021-12-16 16:00:10,501 - new_apply_order.py[line:916]: 校验重试查询第1次
ERROR - 2021-12-16 16:00:11,514 - new_apply_order.py[line:812]: 放款完成或准备失败,目前的订单状态是35
INFO - 2021-12-16 16:00:11,514 - new_apply_order.py[line:813]: {'app': 'fakash', 'loan_order_id': '1471389648496627712', 'mobile': '739641602', 'order_status': 35}
INFO - 2021-12-16 16:00:12,392 - new_apply_order.py[line:643]: 查询loan_order_id成功！
INFO - 2021-12-16 16:00:12,654 - new_apply_order.py[line:781]: 放款回调成功！
INFO - 2021-12-16 16:00:12,654 - new_apply_order.py[line:916]: 校验重试查询第2次
INFO - 2021-12-16 16:00:13,665 - new_apply_order.py[line:810]: 放款完成或准备成功,目前的订单状态是40
INFO - 2021-12-16 16:00:13,665 - new_apply_order.py[line:813]: {'app': 'fakash', 'loan_order_id': '1471389648496627712', 'mobile': '739641602', 'order_status': 40}
INFO - 2021-12-16 16:00:13,666 - new_apply_order.py[line:933]: ===========完成第【1】次进件===========
INFO - 2021-12-16 16:00:13,666 - new_apply_order.py[line:936]: ======完成第【1】次进件,耗时10.71s=====
